site/content/docs/install.md

```diff
@@ -1,444 +1,115 @@
 
-mysql -u root -p openBMP < mysql-openbmp-current.db 
 ```
+ubuntu@docker:~$ docker run hello-world
+Unable to find image 'hello-world:latest' locally
+511136ea3c5a: Pull complete 
+31cbccb51277: Pull complete 
+e45a5af57b00: Pull complete 
+hello-world:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
+Status: Downloaded newer image for hello-world:latest
+Hello from Docker.
+This message shows that your installation appears to be working correctly.
 
-> Use the password for root user that was created when mysql was installed. 
-
+To generate this message, Docker took the following steps:
+ 1. The Docker client contacted the Docker daemon.
+ 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+    (Assuming it was not already locally available.)
+ 3. The Docker daemon created a new container from that image which runs the
+    executable that produces the output you are currently reading.
+ 4. The Docker daemon streamed that output to the Docker client, which sent it
+    to your terminal.
 
-### Run the openbmp server
-
-MySQL should be installed now and it should be running.   OpenBMP is ready to run. 
-
-**openbmpd**   *(normally installed in /usr/bin)*
+To try something more ambitious, you can run an Ubuntu container with:
+ $ docker run -it ubuntu bash
 
+For more examples and ideas, visit:
+ http://docs.docker.com/userguide/
 ```
-  REQUIRED OPTIONS:
-     -c <filename>     Config filename.
-          OR
-     -a <string>       Admin ID for collector, this must be unique for this collector.  hostname or IP is good to use
-
-
-  OPTIONAL OPTIONS:
-     -pid <filename>   PID filename, default is no pid file
-     -l <filename>     Log filename, default is STDOUT
-     -d <filename>     Debug filename, default is log filename
-     -f                Run in foreground instead of daemon (use for upstart)
 
-  OTHER OPTIONS:
-     -v                   Version
-     -h                   Help
 
-  DEBUG OPTIONS:
-     -debugDebug general items
-     -dbgp             Debug BGP parser
-     -dbmp             Debug BMP parser
-     -dmsgbus          Debug message bus
+## Install SNAS Using Docker
+Each docker file contains a readme file, see below:
 
-  DEPRECATED OPTIONS:
-
-       These options will be removed in a future release. You should switch to use the config file.
-     -k <host:port>    Kafka broker list format: host:port[,...]
-                       Default is 127.0.0.1:9092
-     -m <mode>         Mode can be 'v4, v6, or v4v6'
-                       Default is v4.  Enables IPv4 and/or IPv6 BMP listening port
-
-     -p <port>         BMP listening port (default is 5000)
-     -b <MB>           BMP read buffer per router size in MB (default is 15), range is 2 - 128
-     -hi <minutes>     Collector message heartbeat interval in minutes (default is 5 minutes)
-```
+* [All-In-One](/docs/install_aio)
```

Comment:
These links are not rendered correctly due to our side nav and single.html requirement.  You can have them load in the  **docsContent** div by adding a "shortcut" for the link.  The shortcode needs to load "list-group-item" class or we need to add another class that does the same thing as "list-group-item" so that it loads in the docs correctly in the **docsContent** div. 

BTW... we can fix this afterward this PR.